node/test/PublicAPITest.ts

```diff
@@ -33,14 +33,27 @@ class InMemorySessionStore extends SignalClient.SessionStore {
     name: SignalClient.ProtocolAddress
   ): Promise<SignalClient.SessionRecord | null> {
     const idx = name.name() + '::' + name.deviceId();
-    if (this.state.has(idx)) {
+    const serialized = this.state.get(idx);
+    if (serialized) {
       return Promise.resolve(
-        SignalClient.SessionRecord.deserialize(this.state.get(idx))
+        SignalClient.SessionRecord.deserialize(serialized)
       );
     } else {
       return Promise.resolve(null);
     }
   }
+  async getExistingSessions(
+    addresses: SignalClient.ProtocolAddress[]
+  ): Promise<SignalClient.SessionRecord[]> {
+    return addresses.map(address => {
+      const idx = address.name() + '::' + address.deviceId();
+      const serialized = this.state.get(idx);
+      if (!serialized) {
+        throw 'no such session';
```

Comment:
Should the error be consistent with java "no session for " + remoteAddress?

Reply:
I guess. Unlike the Java one, this in-memory implementation is only exposed to tests, but it certainly doesn't hurt to have more descriptive error messages in tests.